### ISSUE TITLE
GD-200: SceneRunner `get_property`  returns error on 'null' value

### ISSUE THREAD

**Issue by clemens-tolboom** [MikeSchulze/gdUnit4#200]
**The used GdUnit4 version:**

4.1.1 (and master)

**The used Godot version:**

v4.0.3.stable.official [5222a99f5]

**OS including version:**

MacOS

**Describe the bug**

Have a nullable var like `var x:RefCounted`
then run `runner.get_property('x')`gives a string back
> "The property 'x' not exist checked loaded scene."

**Steps to Reproduce**

```gdscript
extends Node2D

var x:RefCounted

func _ready():
	printt("X == '{0}'".format([x]))
```
and test
```gdscript
func test_null_value() -> void:
	var runner :GdUnitSceneRunner = scene_runner("res://get_property.tscn")

	var x = runner.get_property("x")
	assert_str(x).is_null()
```

**Minimal reproduction project:**

[issue_200.zip](https://github.com/MikeSchulze/gdUnit4/files/11724983/issue_200.zip)

**Comment by MikeSchulze** [comment]
nice catch 👍

**Comment by clemens-tolboom** [comment]
This is now superseded by #202 thus closed #199
